### PR TITLE
Feature styles changes 2

### PR DESCRIPTION
**Pull request by Melissa-Martinez-XL**
## Ticket

* [Design changes - before release 3](https://www.notion.so/xmartlabs/Design-changes-before-release-3-5123f540df7b4de5a152db9b455fba04?pvs=4)

## Type of change

* [ ] Fix
* [X] Story
* [ ] Chore

## Description of the change

I changed the styles to comply with the comments and added the new header (it is unfinished).

## Screenshot/Execution

https://github.com/xmartlabs/xl-blog/assets/107500715/c8e6619b-5452-4488-938a-7975fced65b4

Captura desde 2023-08-17 17-51-28

## Related PRs
Links to the related PRs

### CODE REVIEW

File: src/components/header-banner/header-banner.module.scss [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  letter-spacing: 0.11rem;
+  text-transform: uppercase;
+  white-space: nowrap;
+  width: 100%;
+  animation: category-move 15s linear infinite;
+  -webkit-animation: category-move 15s linear infinite;
+  justify-content: flex-end;
+  margin-top: 0.2rem;
+
+  p:nth-child(even) {
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply src/components/header-banner/header-banner.module.scss:76]: Done

File: gatsby-browser.js [hunk 2]
```diff
@@ -23,6 +23,10 @@ const GlobalStyles = createGlobalStyle`
         background-color: ${prop => prop.theme.colors.main1};
     }
 
+    body:has(#containerDiv) {
```
Comment (lauta2912) [inline gatsby-browser.js:26]: Why? 😭

Comment (Melissa-Martinez-XL) [reply gatsby-browser.js:26]: jajaj I was trying to do something, but, it didn't work. I forgot to delete it. I'm sorry.

File: src/components/title-blog-index/title-blog-index.jsx [hunk 3]
```diff
@@ -42,6 +42,9 @@ const TitleBlogIndex = ({ data, refIndexTitles, disappearIndex }) => {
       const subContainerHeight = subContainerRef.current.scrollHeight;
       setIsTopArrow(subContainerHeight > viewContainerHeight);
       setIsBottomArrow(subContainerRef.current.scrollTop > 0);
+
+      console.log("Sub" + subContainerHeight)
```
Comment (lauta2912) [inline src/components/title-blog-index/title-blog-index.jsx:46]: to remove

File: src/css/blog-post.module.scss [hunk 4]
```diff
@@ -347,33 +353,40 @@
...
-
 }
 
 @media screen and (max-width: map-get($map: $screen-breakpoint, $key: lg)) {
   .bannerContainer {
     padding: 10rem 27% 20rem 12rem;
   }
+
+  .blogMainImage {
+    transform: translate(191px, -256px);
```
Comment (lauta2912) [inline src/css/blog-post.module.scss:371]: Is this working for all the screen sizes? What's the idea?

File: src/css/typography.css [hunk 5]
```diff
@@ -8,6 +8,11 @@
   src: url("../fonts/Gilroy-Bold.ttf");
 }
 
+@font-face {
+  font-family: "gilroy-semi-bold";
```
Comment (lauta2912) [inline src/css/typography.css:12]: Which PR are you planning solving the font weight issue in?

File: src/pages/categories.module.scss [hunk 6]
```diff
@@ -2,15 +2,16 @@
...
-  grid-template-columns: repeat(auto-fill, minmax(23.4rem, 1fr));
-  gap: 6rem 3.4rem;
+  grid-template-columns: repeat(auto-fill, minmax(28%, 1fr));
+  gap: 4rem 4rem;
 }
 
 @media only screen and (max-width: map-get($map: $screen-breakpoint, $key: sm)) {
   .container {
-    margin: 5rem 0rem;
+    margin: 0r;
```
Comment (lauta2912) [inline src/pages/categories.module.scss:13]: Is this unit needed?

File: src/templates/post.js [hunk 7]
```diff
@@ -78,6 +78,7 @@ const BlogPost = ({ data, children }) => {
 
   const checkWindowOrigin = () => {
     if (typeof window !== 'undefined') {
+      require("smooth-scroll")('a[href*="#"]')
```
Comment (lauta2912) [inline src/templates/post.js:81]: Can't you import it with `import`?

File: src/components/title-blog-index/title-blog-index.jsx (hunk 3 above)
Comment (Melissa-Martinez-XL) [reply src/components/title-blog-index/title-blog-index.jsx:46]: Done

File: src/css/typography.css (hunk 5 above)
Comment (Melissa-Martinez-XL) [reply src/css/typography.css:12]: In tbis PR: https://github.com/xmartlabs/xl-blog/pull/66

File: src/pages/categories.module.scss (hunk 6 above)
Comment (Melissa-Martinez-XL) [reply src/pages/categories.module.scss:13]: Yes, it's needed

File: src/templates/post.js (hunk 7 above)
Comment (Melissa-Martinez-XL) [reply src/templates/post.js:81]: I tried to change it, but I couldn't.

File: src/components/header-banner/header-banner.jsx [hunk 8]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { HeaderCircleIcon } from "../icons";
+
+import * as styles from "./header-banner.module.scss";
+
+const repeatCategories = () => {
+  for (let i = 0; i < 10; i++) {
+    return (
+      <di className={styles.bottomTextSubContainer}>
```
Comment (lauta2912) [inline src/components/header-banner/header-banner.jsx:10]: What's `di`? Did you mean `div` or `dl`?

**Review by lauta2912 — APPROVED** [review]

File: src/components/header-banner/header-banner.jsx (hunk 8 above)
Comment (Melissa-Martinez-XL) [reply src/components/header-banner/header-banner.jsx:10]: No, its div

**Pull request merged**